tests/test_geopedia.py

```diff
@@ -63,8 +63,8 @@ def test_return_type(self):
         self.assertEqual(len(self.data), data_len,
                          "Expected a list of length {}, got length {}".format(data_len, len(self.data)))
 
-    def test_stats(self):
-        self.test_numpy_data(np.array(self.data), exp_min=0, exp_max=255, exp_mean=150.9248, exp_median=255)
+    # def test_stats(self):
```

Comment:
Why is this commented out? If it is not to be used, just remove it.

Reply:
This integration test sometimes fails if Geopedia is under heavy load (like yesterday). We have an issue to improve this but it is not a high priority. Commenting this out was not the best solution therefore I now updated it so that the test is marked with `@pytest.mark.xfail`